chore(release): add release notes for 2.30.5

armory-astrolabe opened the pull request:
Event
```
{
  "armoryServices": [
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Clouddriver",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [
        "chore(cd): update base orca version to 2023.11.06.18.16.22.release-1.30.x (#762)",
        "chore(cd): update base orca version to 2023.11.07.00.08.28.release-1.30.x (#764)",
        "re-trigger build (#766)"
      ],
      "currentVersion": "2.30.5",
      "name": "Armory Orca",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Igor",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Rosco",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Gate",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Front50",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Fiat",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Armory Kayenta",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Terraformer™",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [
        "fix: Upgrade grpc-netty-shaded to address the service initialization failure issue. (#647)"
      ],
      "currentVersion": "2.30.5",
      "name": "Armory Echo",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [],
      "currentVersion": "2.30.5",
      "name": "Dinghy™",
      "previousVersion": "2.30.4"
    },
    {
      "commitMessages": [
        "chore(cd): update base deck version to 2023.0.0-20231024161913.release-1.30.x (#1364)"
      ],
      "currentVersion": "2.30.5",
      "name": "Armory Deck",
      "previousVersion": "2.30.4"
    }
  ],
  "armoryVersion": "2.30.5",
  "ossServices": [
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Clouddriver",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [
        "fix(artifacts): Parent and child pipeline artifact resolution (backport #4575) (#4581)",
        "fix(artifacts): Automated triggers with artifact constraints are broken if you have 2 or more of the same type (backport #4579) (#4586)"
      ],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Orca",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Igor",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Rosco",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Gate",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Front50",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Fiat",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Kayenta",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Echo",
      "previousVersion": "1.30.0"
    },
    {
      "commitMessages": [
        "fix: Updating Lambda functions available Runtimes (#10055) (#10056)"
      ],
      "currentVersion": "1.30.4",
      "name": "Spinnaker Deck",
      "previousVersion": "1.30.0"
    }
  ],
  "ossVersion": "1.30.4",
  "prerelease": false,
  "stack": {
    "artifactSources": {
      "dockerRegistry": "docker.io/armory"
    },
    "dependencies": {
      "redis": {
        "commit": null,
        "version": "2:2.8.4-2"
      }
    },
    "services": {
      "clouddriver": {
        "commit": "767aa739e9c38d2ec7822e9e9a1838b69a56d4c0",
        "version": "2.30.5"
      },
      "deck": {
        "commit": "b14f5660fbe9cde883dfaf25442a1e8cfe0c27b0",
        "version": "2.30.5"
      },
      "dinghy": {
        "commit": "a3b59d9e4b810cc968d0f5e8e8370e1670574768",
        "version": "2.30.5"
      },
      "echo": {
        "commit": "ea2081903be070da4b217a4c54d070f1503e6d00",
        "version": "2.30.5"
      },
      "fiat": {
        "commit": "5d44e1f53d2f33b17f8decfb057a18cfe4b6fa08",
        "version": "2.30.5"
      },
      "front50": {
        "commit": "5d9bb31f65f96087be30ce96cea1b6d481a6bef4",
        "version": "2.30.5"
      },
      "gate": {
        "commit": "5758316afba1260d2012730c471fd461819e7f39",
        "version": "2.30.5"
      },
      "igor": {
        "commit": "3123451525458f96548859b9bf2b15c89810f577",
        "version": "2.30.5"
      },
      "kayenta": {
        "commit": "1d2f5193ec681b5122fe7c34da6bbd569da8e0b8",
        "version": "2.30.5"
      },
      "monitoring-daemon": {
        "commit": null,
        "version": "2.26.0"
      },
      "monitoring-third-party": {
        "commit": null,
        "version": "2.26.0"
      },
      "orca": {
        "commit": "fdac510ea4f54b33abafbc840d7380100c6ab689",
        "version": "2.30.5"
      },
      "rosco": {
        "commit": "7907c80fd9ca1d956d5927a3e190617671b7e012",
        "version": "2.30.5"
      },
      "terraformer": {
        "commit": "249e7be18074af100212ddd554d9fb35afd65873",
        "version": "2.30.5"
      }
    },
    "timestamp": "2023-11-08 19:09:49",
    "version": "2.30.5"
  }
}
```

armory-astrolabe committed:
chore(release): add release notes for 2.30.5

mknous committed:
Update armoryspinnaker_v2-30-5.md

First cut at the release notes. @dogonthehorizon can you check to see if I can remove the first two known issues related to artifact binding as I think they were addressed by the PRs we did in this release.

For highlighted updates, there were three changes I called out in this release for Echo, Deck and Orca.

mknous commented:
I believe the notes are good. We need to check and see if the artifact binding work we did resolved two of the known issues that are called out.

dogonthehorizon commented on content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-30-5.md:
```diff
@@ -0,0 +1,292 @@
+
+  - chore(cd): update base orca version to 2023.11.06.18.16.22.release-1.30.x (#762)
+  - chore(cd): update base orca version to 2023.11.07.00.08.28.release-1.30.x (#764)
+  - re-trigger build (#766)
```
```suggestion
```

mknous committed:
Update content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-30-5.md

Co-authored-by: Fernando Freire <fernando.freire@armory.io>

mknous committed:
Update armoryspinnaker_v2-30-5.md

Removing the two known issues related to Artifact binding. Per @nemesisOsorio if the PRs got in from 519 these can be removed.
